tzdata/supplemental/timezoneTypes.txt

```diff
@@ -236,6 +236,7 @@ timezoneTypes:table(nofallback){
             "America:Cayman"{"kygec"}
             "America:Chicago"{"uschi"}
             "America:Chihuahua"{"mxchi"}
+            "America:Ciudad_Juarez"{"mxcjs"}
```

Comment:
```suggestion
            "America:Ciudad_Juarez"{"mxcjz"}
```

Reply:
Yoshito replied on the CLDR PR that mxcjs is correctly based on another standard: https://github.com/unicode-org/cldr/pull/2583#discussion_r1036524022